kotlin-semver/src/jsMain/generated/semver/RangeOptions.kt

```diff
@@ -0,0 +1,10 @@
+// Generated by Karakum - do not modify it manually!
+
+package semver
+
+import semver.classes.SemVer
+
+@kotlinx.js.JsPlainObject
+external interface RangeOptions : Options {
+var includePrerelease: Boolean?
```

Comment:
Could you please format generated sources?

Reply:
I would be delighted to, the strange readability constantly bugged me.
But I thought the generated sources should not be touched?
Also, if you want the generated sources to be uniformly and nicely formatted, maybe after generation some automatica formatting should be happening, for example by sending the files through ktlint?

Reply:
> maybe after generation some automatica formatting should be happening, for example by sending the files through ktlint?

Automatic tool - under discussion.
For now - formatting + cleanup in IDEA is what we use.